.github/workflows/build.yml

```diff
@@ -12,15 +12,16 @@ jobs:
               with:
                   node-version: 20
             - name: Checkout Repository
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
               with:
                   token: ${{ secrets.GH_TOKEN }}
             - name: Install modules
               uses: pnpm/action-setup@v2
               with:
                   version: 8
-            - name: Install dependencies
-              run: pnpm install
+                  run_install: |
+                      - recursive: true
+                        args: [--frozen-lockfile, --strict-peer-dependencies]
```

Comment:
I guess the peer deps were the problem?

Reply:
i think recursive was the solution and adding cjs-builder to deps of registry also helped.